tests/unit/backend/data/test_LocalDataService.py

```diff
@@ -364,7 +495,21 @@ def test_writeCalibrationState():
             calibration = Calibration.parse_raw(f.read())
         localDataService.writeCalibrationState("123", calibration)
         assert os.path.exists(Resource.getPath("outputs/v_1/CalibrationParameters.json"))
-        shutil.rmtree(Resource.getPath("outputs/v_1"))
+        # shutil.rmtree(Resource.getPath("outputs/v_1"))
+
+    def test_writeNormalizationState():
+        localDataService = LocalDataService()
+        localDataService._generateStateId = mock.Mock()
+        localDataService._generateStateId.return_value = ("123", "456")
+        localDataService._constructCalibrationStatePath = mock.Mock()
+        localDataService._constructCalibrationStatePath.return_value = Resource.getPath("outputs/")
+        localDataService._getCurrentNormalizationRecord = mock.Mock()
+        localDataService._getCurrentNormalizationRecord.return_value = Normalization.construct({"name": "test"})
+        with Resource.open("/inputs/normalization/NormalizationParameters.json", "r") as f:
+            normalization = Normalization.parse_raw(f.read())
+        localDataService.writeNormalizationState("123", normalization)
+        assert os.path.exists(Resource.getPath("outputs/v_1/CalibrationParameters.json"))
+        # shutil.rmtree(Resource.getPath("outputs/v_1"))
```

Comment:
I think all of these shutil.rmtree statements are necessary, or else the /outputs/ directory will fill up with v1, v2, v3, etc.  After today's standup, it might be worth using a temporary directory for this test suite,